custom_components/elasticsearch/__init__.py

```diff
@@ -149,6 +145,14 @@ async def async_migrate_entry(hass: HomeAssistantType, config_entry: ConfigEntry
         if CONF_INDEX_MODE not in new:
             new[CONF_INDEX_MODE] = INDEX_MODE_LEGACY
 
+        CONF_ILM_MAX_SIZE = "ilm_max_size"

```

Comment:
Can we add a test for this config migration to `tests/test_init.py`? There are examples at the bottom of that file that you can use as a template.